src/main/kotlin/tay/example/manage_stock_of_book/model/Category.kt

```diff
@@ -0,0 +1,30 @@
+package tay.example.manage_stock_of_book.model
+
+import org.hibernate.annotations.CreationTimestamp
+import org.hibernate.annotations.UpdateTimestamp
+import java.time.LocalDateTime
+import javax.persistence.*
+
+
+@Entity
+@Table(name = "categories")
+data class Category(
+        @Id
+        @GeneratedValue(strategy = GenerationType.AUTO)
+        @Column(name = "id")
+        var id: Long = 0,
+
+        @Column(name = "name")
+        var name: String,
+
+        @Column(name = "description")
+        var description: String,
+
+        @CreationTimestamp
+        @Column(name ="createdAt")
```

Comment:
Please use underscore name style for naming column in database
@Column(name = "created_at")